youtube_dl/extractor/bbc.py

```diff
@@ -761,8 +764,13 @@ class BBCIE(BBCCoUkIE):
         'only_matching': True,
     }, {
         # custom redirection to www.bbc.com
+        # video with window.__INITIAL_DATA__
         'url': 'http://www.bbc.co.uk/news/science-environment-33661876',
-        'only_matching': True,
+        'id': 'p02xzws1',
+        'title': "Pluto may have 'nitrogen glaciers'",
+        'description': "Pluto could have glaciers of nitrogen ice, new photographs from Nasa's New Horizons probe suggest.",
+        'thumbnail': r're:https?://.+/.+\.jpg',
+        'timestamp': 1437785037,
```

Comment:
This must be inside a dict corresponding to `info_dict` key. `python -W error .\test\test_download.py TestDownload.test_BBC_13` must succeed.

Reply:
Obvs, stupid cut+paste error, thanks.